### PR TITLE
Added settings required to build access_service data site template

### PR DESCRIPTION
**Pull request by roxigonzalez**
Now it's added the `access_service` data
Screenshot 2021-06-10 at 15 18 06

### CODE REVIEW

File: lib/openactive/dataset_site/settings.rb [hunk 1]
```diff
@@ -94,6 +102,20 @@ def booking_service
         )
       end
 
+      def access_service
+        OpenActive::Models::WebAPI.new(
+          name: 'Open Booking API',
+          description: "API that allows for seamless booking experiences to be created for #{data_feed_descriptions.to_sentence} available from #{organisation_name}",
```
Comment (nickevansuk) [inline lib/openactive/dataset_site/settings.rb]: `data_feed_descriptions.to_sentence` should also be converted to lower case

**Review by nickevansuk — CHANGES_REQUESTED** [review]

File: lib/openactive/dataset_site/settings.rb (hunk 1 above)
Comment (roxigonzalez) [reply lib/openactive/dataset_site/settings.rb]: Done

File: lib/openactive/dataset_site/settings.rb [hunk 2]
```diff
@@ -94,6 +102,20 @@ def booking_service
         )
       end
 
+      def access_service
+        OpenActive::Models::WebAPI.new(
+          name: 'Open Booking API',
+          description: "API that allows for seamless booking experiences to be created for #{data_feed_descriptions.to_sentence} available from #{organisation_name}".downcase,
```
Comment (nickevansuk) [inline lib/openactive/dataset_site/settings.rb]:
```suggestion
          description: "API that allows for seamless booking experiences to be created for #{data_feed_descriptions.to_sentence.downcase} available from #{organisation_name}",
```

Comment (nickevansuk) [reply lib/openactive/dataset_site/settings.rb]: Would this work instead? It's only the `data_feed_descriptions.to_sentence` that needs to be lower cased...

**Review by nickevansuk — CHANGES_REQUESTED** [review]

**Pull request merged**